Diffusion/Pipeline/PipelineLoader.swift

```diff
@@ -64,11 +62,21 @@ class PipelineLoader {
 }
 
 extension PipelineLoader {
+    // Note: this function is currently unused. Marked for deletion. -- dolmere
     static func removeAll() {
-        try? models.delete()
+        let enumerator = FileManager.default.enumerator(at: models, includingPropertiesForKeys: nil)
+
+        while let fileURL = enumerator?.nextObject() as? URL {
+            do {
+                try FileManager.default.removeItem(at: fileURL)
+            } catch {
+                print("Failed to delete: \(fileURL), error: \(error.localizedDescription)")
+            }
+        }
```

Comment:
```suggestion
        do {
            try FileManager.default.removeItem(at: models)
        } catch {
            print("Failed to delete: \(models), error: \(error.localizedDescription)")
        }
```

I think there's no need to enumerate, it's ok to delete the parent folder as it will be recreated when it's needed.